package.json

```diff
@@ -34,7 +34,7 @@
     "node": ">=10.0.0"
   },
   "browserslist": [
-    "last 2 versions",
```

Comment:
This has no affect on CI, but it does reflect our support intention for browsers like Firefox and Chrome.